src/components/glossaryComponents/collabsibleTable/collapsibleTable.js

```diff
@@ -0,0 +1,292 @@
+import { Link } from "react-router-dom";
+import Accordion from 'react-bootstrap/Accordion'
+import EnvironmentalRating from "../../stockVisualisationComponents/ESGRatingSliders/EnvironmentalRating";
```

Comment:
I know you said you were going to add this in down the line, but just make sure that if we're not adding it in to remove it!